sdl2promela/stop_condition/translator.py

```diff
@@ -1234,6 +1364,20 @@ def _translate_basic_statements(
     for never in input_model.never_statements:
         builder.withAlternative(_generate_never_alternative(never, context))
 
+    if context.observers_with_error:
+        for observer in context.observers_with_error:
+            states = context.processes[observer].errorstates
+            builder.withAlternative(
+                _create_never_alternative_for_observer(states, observer)
+            )
+
+    if context.search_for_observer_success and context.observers_with_success:
```

Comment:
This requires a comment for future generations

Reply:
added a comment block